### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by GrebilCruz**
added Media querry in style.css
Added work section with desktop version
changed contact form for desktop version
changed some works section with reverser direction
Added style for desktop version
added navigation bar/fixed in all the desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -235,7 +251,7 @@ body {
 
 .project {
   width: fit-content;
-  margin-top: 20px;
+  margin-top: 60px;
```
Comment (ShadyShawkat) [inline style.css]: - [ ] Kindly remove or reduce the `margin-top` in this class to match the Figma design. 
image

File: style.css [hunk 2]
```diff
@@ -473,3 +514,320 @@ img.arrow {
...
+}
+
+#grid_container__desktop {
+  display: none;
+}
+
+/* <<<<<<<<<<<DESKTOP VERSION>>>>>>>>>>>>>>>>> */
+
+@media only screen and (min-width: 900px) {
+  .nav_bar {
```
Comment (ShadyShawkat) [inline style.css]: - [ ] Please make sure to use `768px` not `900px` as the minimum width for desktop view, [as per requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=Desktop%3A%20768px%20or%20wider).

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @GrebilCruz, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly remove the `margin-left` and `margin-right` from this class, as it's causing a horizontal scroll on mobile view width `375px`.
https://github.com/GrebilCruz/Portfolio/blob/a29c4962f85c4a6c798f9aa2272faeef72da86c6/style.css#L142-L153

- [ ] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @GrebilCruz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**